### PR TITLE
fix(tasks): update tasks panel z index

### PR DESCRIPTION
**Pull request by pedrobonamin**
### Description
Updates tasks sidebar z index to sit on top of the rest of elements.

### CODE REVIEW

File: packages/sanity/src/core/tasks/plugin/TasksStudioActiveToolLayout.tsx [hunk 1]
```diff
@@ -77,6 +77,7 @@ function TasksStudioActiveToolLayoutInner(props: ActiveToolLayoutProps) {
         {isOpen && (
           <SidebarMotionLayer
             animate="visible"
+            zOffset={1000}
```
Comment (RitaDias) [inline packages/sanity/src/core/tasks/plugin/TasksStudioActiveToolLayout.tsx:80]: @pedrobonamin 👋 Quick question: which issue is this trying to solve? 👀

Comment (pedrobonamin) [reply packages/sanity/src/core/tasks/plugin/TasksStudioActiveToolLayout.tsx:80]: 👋🏻 
Screenshot 2024-04-17 at 15 47 19

Comment (pedrobonamin) [reply packages/sanity/src/core/tasks/plugin/TasksStudioActiveToolLayout.tsx:80]: Some elements are showing on top of the tasks sidebar.

**Review by RitaDias — APPROVED** [review]
🚢

**Pull request merged**